docs/api-docs/headless/end-to-end-guide/graphql.mdx

```diff
@@ -134,7 +134,7 @@ Make note of one of the cart's `entityId` displayed in the response. You will ne
     mutation createCartSimple($createCartInput: CreateCartInput!) {
       cart {
         createCart(input: $createCartInput) {
-          cart {
+          cart(entityId: $entityId) {
```

Comment:
The addition of an "argument" here is not correct.

When you are using a "dollar sign" variable like $entityId here, it is a reference to a variable that has been passed into the query/mutation:

```
mutation createCartSimple($createCartInput: CreateCartInput!) {
```

$entityId is not a variable being passed in on the line above, so there's nothing to reference. And, indeed, this is not relevant to the createCartSimple mutation; it's creating a cart for the first time, there is no existing ID to pass in.  This mutation should remain as it was.  (The key is that the user will need to _save_ the entityId that is being queried on the next line.)

Reply:
that makes sense